image-sync/on-demand/sync.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+set -euo pipefail
+
+# only run within EV2 for now
+# RH dev envs have the regular component sync in place to make images available
+if [ -z "${EV2:-}" ]; then
```

Comment:
I imagine you have `shellcheck` but if not, I suggest it. Prefer `[[` over `[` unless you want to rip your hair out over POSIX compliance.

Reply:
ty. fixed